civihr_employee_portal/templates/civihr-employee-portal-custom-reports.tpl.php

```diff
@@ -105,6 +105,9 @@ class="btn btn-lg btn-secondary-outline text-uppercase">
             </div>
         </footer>
     </section>
+    <div data-graph-section="summary">
```

Comment:
You have to choices here, I think:

1. You stick with creating the HTML via javascript, but then please see if you can reuse some of the code already written for it, given that we're already creating a table in `reports.js`
2. You use an hidden template that you clone and populate in JS (preferred). You can see how it's done in the `cloneButtonTpl` function